src/helpers/chevrotain.ts

```diff
@@ -0,0 +1,12 @@
+import { createToken, ITokenConfig } from 'chevrotain';
```

Comment:
I don't believe this file is being used

Reply:
Oh, I see it in `sparql/keywords` now, just not in `sms/tokens`. It might be better to name it something that isn't so close to the chevrotain module. If you imagine this file also holding other token helpers maybe `tokens`, or otherwise `createKeyword`

Reply:
yes, that can work. this file is intended to be a chevrotain helper

Reply:
We do have other files containing methods and types that are essentially "chevrotain helpers" in that provide additional functionality from building around the chevrotain package. For example, `helpers/cst` or `helpers/types` indicates what part of chevrotain we're "helping" or answers the question `what are these helpers for?` Having a file in the helpers called `chevrotainHelper`, especially alongside the existing helper files, is not descriptive as to what kind of functionality this file actually covers. Can you rename this file to be something more specific?

Reply:
yes, just took a look. Your suggestion for this file was `tokens`. We can use that.

It is also not entirely clear what kind of helpers `helpers/cst`, and `helpers/types` are. These 3 files are better suited in a folder named `chevrotain` .